website/src/pages/docs/guides/vanilla-typescript.mdx

```diff
@@ -181,6 +181,7 @@ We can build a simple wrapper around `fetch` that takes a `TypedDocumentString`
 returns a typed response.
 
 ```typescript filename="src/graphql/execute.ts"
+import { ExecutionResult } from 'graphql'
```

Comment:
I think most readers would be on the later versions of TypeScript which allows `import type`. I think this is better in general 🙂

```suggestion
import type { ExecutionResult } from 'graphql'
```

Reply:
Adjusted @eddeee888